src/server/static/css/main.css

```diff
@@ -132,6 +132,34 @@ header h1:before {
   margin: 20px auto;
 }
 
+.container-wrapper {
```

Comment:
While I appreciate that this is a better way to split the columns, the Bootstrap grid solution works on small screens and this does not. Do we want to remove the auto-stacking functionality that Bootstrap provides?

Reply:
For the record, I'm in favor of keeping the Bootstrap grid and maintaining compatibility with smaller screens.

Reply:
Hm this PR still does some nice stacking for me, so not sure in what contexts it will break: ![image](https://user-images.githubusercontent.com/21347592/76986855-79584a80-68ff-11ea-96e5-5c2c0d1880c5.png)

Also, this view isn't too bad: 
![image](https://user-images.githubusercontent.com/21347592/76986909-8d03b100-68ff-11ea-8ece-dd7c21804169.png)

Reply:
Oh, right. There's an idea. Why don't I try it on my machine first? (and why don't I look at the context for the CSS rules below.)

Yeah, this checks out now. I think we should have some sort of max-width on the container, but not gonna worry about that right now. I'll update the PR with compatible styles for independent scroll.